domains/tespandj.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "talanngaï",
```

```suggestion
    "username": "tespandj",
```

use your github username